dali_tf_plugin/dali_tf_plugin_install_tool.py

```diff
@@ -174,9 +188,11 @@ def install(self):
             + plugin_src + ' -o ' + lib_path + ' -fPIC ' + dali_cflags + ' ' \
             + tf_cflags + ' ' + cuda_cflags + ' ' + dali_lflags + ' ' + tf_lflags + ' ' \
             + cuda_lflags + ' -O2'
-        print("Build command:\n\n " + cmd + '\n\n')
+        print("Build DALI TF library:\n\n " + cmd + '\n\n')
         subprocess.check_call(cmd, cwd=self.src_path, shell=True)
 
+        # Remove DALI stub
+        rmtree(tmpdir)
```

Comment:
Why not use `tempfile.TemporaryDirectory` which will clean itself?

Reply:
done